specs/mrtcore/PrimaryLanguageOverride/PrimaryLanguageOverride.md

```diff
@@ -0,0 +1,190 @@
+
+`GetLanguagesForUser(User)` method from `Windows.Globalization.ApplicationLanguages` is being
+intentionally left off due to not being relevant for WinAppSDK.
+
+# API Pages
+
+## ApplicationLanguages class
+
+Specifies the language-related preferences that the app can use and maintain. This class is based on
+OS
+[Windows.Globalization.ApplicationLanguages](https://learn.microsoft.com/uwp/api/windows.globalization.applicationlanguages)
+class. You should prefer this type for WinAppSDK apps over the OS one.
+
+# Remarks
+
+The languages referenced in this class are represented by
+[BCP-47](https://tools.ietf.org/html/bcp47) language tags. Each of these is a language identifier
+and not a locale name, although the two structurally can be the same. As a language tag, "en-US"
+means American English (or the dialect of English spoken in the United States). It makes no claims
+about the actual location of the speakers (though many of them do reside in the US) and no other
+claims about conventions that are used in the US such as the measurement system or currency.
+
+Example:
+
+```c#
+Microsoft.Windows.ApplicationModel.Resources.ApplicationLanguages.PrimaryLanguageOverride = "en-US";
+```
+
+Language tags support the Unicode extensions "ca-" and "nu-". (See Unicode Key/Type Definitions.)
+Note that these extensions can affect the numeral system or calendar used by globalization objects.
+
+## ApplicationLanguages.Languages property
+
+Gets a ranked list of current runtime language values preferred by the user.
+
+# Property Value
+
+[IReadOnlyList](https://learn.microsoft.com/dotnet/api/system.collections.generic.ireadonlylist-1)<[String](https://learn.microsoft.com/dotnet/api/system.string)>
+
+A computed list of languages that merges the app's declared supported languages
+(ApplicationLanguages.ManifestLanguages) with the user's ranked list of preferred languages.
+
+Note: This property returns the same values as the language list exposed by
+[Windows.Globalization.ApplicationLanguages.Languages](https://learn.microsoft.com/uwp/api/windows.globalization.applicationlanguages.languages)
+property.
+
+# Remarks
+
+At runtime, the list of languages for which your app has declared support (the app manifest language
+list) is compared with the list of languages for which the user has declared a preference (the user
+profile language list). The app runtime language list is set to this intersection (if the
+intersection is not empty), or to just the app's default language (if the intersection is empty).
+For more detail, see the
+[App runtime language list](https://learn.microsoft.com/windows/apps/design/globalizing/manage-language-and-region#app-runtime-language-list)
+section in
+[Understand user profile languages and app manifest languages](https://learn.microsoft.com/windows/apps/design/globalizing/manage-language-and-region).
+
+## ApplicationLanguages.ManifestLanguages property
+
+Gets the app's declared list of supported languages.
```

Comment:
What if you have no manifest (unpackaged app)?

Reply:
RECOMMEND: 
1. Make this return an empty list instead of throwing an exception.
2. Update the doc here to state that it returns an empty list.